app/views/components/docs/banner.yml

```diff
@@ -5,7 +5,8 @@ accessibility_criteria: |
   The banner must:
 
   - be visually distinct from other content on the page
-  - have a contrast ratio higher than 4.5:1 against the background colour to meet WCAG AA
+  - have an accessible name that describes the banner as a notice
+  - have a text contrast ratio higher than 4.5:1 against the background colour to meet [WCAG AA](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast)
```

Comment:
We avoid confusion over the definition of large text by sticking with 4.5:1 for everything.